fi.json

```diff
@@ -1720,23 +1720,23 @@
         }
     },
     "nouns": {
-        "word-with-count": "{{count}} word",
-        "word-with-count_plural": "{{count}} words",
-        "character-with-count": "{{count}} character",
-        "character-with-count_plural": "{{count}} characters",
-        "link-with-count": "{{count}} link",
-        "link-with-count_plural": "{{count}} links",
-        "file-with-count": "{{count}} file",
-        "file-with-count_plural": "{{count}} file",
-        "folder-with-count": "{{count}} folder",
-        "folder-with-count_plural": "{{count}} folders",
-        "backlink-with-count": "{{count}} backlink",
-        "backlink-with-count_plural": "{{count}} backlinks",
-        "site-with-count": "{{count}} site",
-        "site-with-count_plural": "{{count}} sites",
-        "plugin-with-count": "{{count}} plugin",
-        "plugin-with-count_plural": "{{count}} plugins",
-        "theme-with-count": "{{count}} theme",
-        "theme-with-count_plural": "{{count}} themes"
+        "word-with-count": "{{count}} sana",
+        "word-with-count_plural": "{{count}} sanaa",
+        "character-with-count": "{{count}} merkki",
+        "character-with-count_plural": "{{count}} merkkiä",
+        "link-with-count": "{{count}} linkki",
+        "link-with-count_plural": "{{count}} linkkiä",
+        "file-with-count": "{{count}} tiedosto",
+        "file-with-count_plural": "{{count}} tiedostoa",
+        "folder-with-count": "{{count}} kansio",
+        "folder-with-count_plural": "{{count}} kansiota",
+        "backlink-with-count": "{{count}} referoiva linkki",
+        "backlink-with-count_plural": "{{count}} referoivaa linkkiä",
+        "site-with-count": "{{count}} sijainti",
+        "site-with-count_plural": "{{count}} sijaintia",
+        "plugin-with-count": "{{count}} liitännäinen",
```

Comment:
I've used the word "lisäosa" for plugin and I think we should only use 1 translation for it. We should either change this to "lisäosa" or change the other places to use "liitännäinen". Both are fine in my opinion, you can decide which we should go with.

Reply:
"Lisäosa" is shorter, I'll fix this PR up by today